src/shared/components/Tabs/TabElement.tsx

```diff
@@ -32,16 +47,9 @@ const TabElement: React.FunctionComponent<TabElementProps> = ({
         <p className="description fade">{description}</p>
       </div>
       <div className={'edit-button-container'}>
-        <Button
-          type="primary"
-          icon="edit"
-          size="small"
-          onClick={e => {
-            onEditClick(id);
-            e.stopPropagation();
-          }}
-          style={buttonStyle}
-        />
+        {studioPermissionsPath
+          ? resourcesWritePermissionsWrapper(editButton, studioPermissionsPath)
```

Comment:
Lets lift this up into a prop from the parent container because this has logic that doesn't belong in a component (stateful app stuff, specifically the AcessControl using nexusClient)